get_involved.md

```diff
@@ -5,6 +5,20 @@ permalink: /get_involved/
 toc: false
 ---
 
+## Upcoming Events
+
+**Bug Triage** - every Tuesday and Friday in IRC. Come and participate in real-time. See the
```

Comment:
This one seems different than the others in that it's a meeting. Can we have a `## Meetings` section added with just this one in it?

Reply:
Also what about making this the authoritative source on the date/times rather than sending people off of this page?

Reply:
I'll a meetings section above the Upcoming Events. I'll also add the time for triage.